leapp/exceptions.py

```diff
@@ -63,6 +63,13 @@ def __init__(self, path):
             'Multiple actors found in {path}. Inspection failed'.format(path=path))
 
 
+class MultipleConfigActorsError(LeappError):
```

Comment:
Where is `WorkflowConfigNotAvailable`?

Reply:
Oops, not saved vscode file. Will fix that first thing tomorrow...